src/mono/wasm/runtime/driver.c

```diff
@@ -774,6 +776,12 @@ mono_wasm_assembly_get_entry_point (MonoAssembly *assembly)
 
 	end:
 	MONO_EXIT_GC_UNSAFE;
+	if (auto_insert_breakpoint)
+	{
+		MonoAssemblyName *aname = mono_assembly_get_name (assembly);
+		const char *name = mono_assembly_name_get_name (aname);
```

Comment:
Or we could even skip calling `mono_wasm_set_..`, `if !name`?

Reply:
done